docs/guides/google-auth-hook.md

```diff
@@ -0,0 +1,227 @@
+---
+id: google_auth
+title: Google Auth Hook
+---
+
+This part of the tutorial shows how to create a sign-in flow for users and integrate with Unleash server project. The implementation assumes that I am working in `localhost` with `4242` port.
+
+This is a simple `index.js` server file.
+```javascript
+const unleash = require('unleash-server');
+
+if (process.env.DATABASE_URL_FILE) {
+  options.databaseUrl = fs.readFileSync(process.env.DATABASE_URL_FILE);
+}
+
+unleash.start(options).then(unleash => {
+  console.log(`Unleash started on http://localhost:${unleash.app.get('port')}`);
+});;
+```
+
+### Creating a web application client ID
+
+1. Go to the credentials section in the [Google Cloud Platform Console](https://console.cloud.google.com/apis/credentials?_ga=2.77615956.-1991581217.1542834301).
```

Comment:
LOW: You should probably remove those `_ga` attributes.

Reply:
Ups! :sweat_smile: